src/net/server/Server.java

```diff
@@ -1554,6 +1556,75 @@ private boolean isFirstAccountLogin(Integer accId) {
         }
     }
     
+    private static void applyAllNameChanges() {
+        try (Connection con = DatabaseConnection.getConnection();
+                PreparedStatement ps = con.prepareStatement("SELECT * FROM namechanges WHERE completionTime IS NULL")) {
+            ResultSet rs = ps.executeQuery();
+            List<Pair<String, String>> changedNames = new LinkedList<Pair<String, String>>(); //logging only
+            con.setAutoCommit(false);
+            while(rs.next()) {
+                int nameChangeId = rs.getInt("id");
+                int characterId = rs.getInt("characterId");
+                String oldName = rs.getString("old");
+                String newName = rs.getString("new");
+                MapleCharacter.doNameChange(con, characterId, oldName, newName, nameChangeId);
+                changedNames.add(new Pair<String, String>(oldName, newName));
+            }
+            con.setAutoCommit(true);
+            //log
+            for(Pair<String, String> namePair : changedNames) {
+                FilePrinter.print(FilePrinter.CHANGE_CHARACTER_NAME, "Name change applied : from \"" + namePair.getLeft() + "\" to \"" + namePair.getRight() + "\" at " + Calendar.getInstance().getTime().toString());
+            }
+        } catch(SQLException e) {
+            e.printStackTrace();
+        }
+    }
+    
+    private static void applyAllWorldTransfers() {
+        try (Connection con = DatabaseConnection.getConnection();
+                PreparedStatement ps = con.prepareStatement("SELECT * FROM worldtransfers WHERE completionTime IS NULL")) {
+            ResultSet rs = ps.executeQuery();
+            List<Integer> removedTransfers = new LinkedList<Integer>();
+            while(rs.next()) {
+                int nameChangeId = rs.getInt("id");
+                int characterId = rs.getInt("characterId");
+                int oldWorld = rs.getInt("from");
+                int newWorld = rs.getInt("to");
+                String reason = MapleCharacter.checkWorldTransferEligibility(con, characterId, oldWorld, newWorld); //check if character is still eligible
+                if(reason != null) {
+                    removedTransfers.add(nameChangeId);
+                    FilePrinter.print(FilePrinter.WORLD_TRANSFER, "World transfer cancelled : Character ID " + characterId + " at " + Calendar.getInstance().getTime().toString() + ", Reason : " + reason);
+                    try (PreparedStatement delPs = con.prepareStatement("DELETE FROM worldtransfers WHERE id = ?")) {
+                        delPs.setInt(1, nameChangeId);
+                        delPs.executeUpdate();
+                    } catch(SQLException e) { e.printStackTrace(); }
```

Comment:
Why are some exception catches compacted? Also all exceptions from this PR should be logged.

Reply:
Was just to save on useless lines. Added exception logging to `doWorldTransfer()` and `doNameChange()`.